src/main/java/io/github/thebusybiscuit/slimefun4/api/events/MultiBlockBreakEvent.java

```diff
@@ -0,0 +1,83 @@
+package io.github.thebusybiscuit.slimefun4.api.events;
+
+import java.util.List;
+
+import javax.annotation.Nonnull;
+import javax.annotation.Nullable;
+import javax.annotation.ParametersAreNonnullByDefault;
+
+import org.bukkit.block.Block;
+import org.bukkit.entity.Player;
+import org.bukkit.event.Cancellable;
+import org.bukkit.event.Event;
+import org.bukkit.event.HandlerList;
+
+/**
+ * This {@link Event} is called when a {@link Player} breaks multiply blocks.
+ * 
+ * @author TheBusyBiscuit
+ *
+ */
+public class MultiBlockBreakEvent extends Event implements Cancellable {
```

Comment:
That name is completely misleading...
Slimefun has things called "MultiBlocks", so this is VERY misleading...

Reply:
have you any good other name idea like ``BlocksBreakEvent``?

Reply:
Okay